json_object.c

```diff
@@ -933,7 +933,21 @@ int json_c_set_serialization_double_format(const char *double_format, int global
 #endif
 		if (global_serialization_float_format)
 			free(global_serialization_float_format);
-		global_serialization_float_format = double_format ? strdup(double_format) : NULL;
+		if (double_format)
+		{
+			char *p = strdup(double_format);
+			if (p == NULL)
+			{
+				_json_c_set_last_err("json_c_set_set_serialization_double_format: "
```

Comment:
typo: doubled `_set_set_`

(also below ... and also back in https://github.com/json-c/json-c/pull/755/files#diff-3dbecd622f44dd0c1d2cc6363ed0f5f5b13d0cc912f1d7580ebb2875dffab023R948 fwiw)

Reply:
Fixed, thank you! I have a talent to include typos in a message which I am about to copy...